src/components/radios/template.njk

```diff
@@ -61,7 +61,7 @@
       }) | indent(6) | trim }}
     </div>
     {% if item.conditional %}
-      <div class="govuk-radios__conditional" id="{{ conditionalId }}">
+      <div class="govuk-radios__conditional{% if not item.checked %} is-hidden{% endif %}" id="{{ conditionalId }}">
```

Comment:
For users that do not use macros, how do we document this?

Reply:
Actually I realised that this still wouldn't work because we need to we do `.js-enabled .js-hidden` in CSS so that style doesn't kick in until JS has loaded 😞 I'll keep on thinking.

Reply:
That's currently how the JavaScript logic works. The logic in the template is for the initial render.

I think this might not be a big issue since the example we use in the Design System website has a pre-checked input which means you'd see it in the generated HTML.